Configuration/PyReleaseValidation/python/relval_standard.py

```diff
@@ -448,26 +448,52 @@
 workflows[139.005] = ['',['AlCaPhiSym2021','RECOALCAECALPHISYMDR3','ALCAECALPHISYM']]
 
 ### run3 (2022) ###
-workflows[140.001] = ['',['RunMinimumBias2022B','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
-workflows[140.002] = ['',['RunZeroBias2022B','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
-workflows[140.003] = ['',['RunJetHT2022B','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
-workflows[140.004] = ['',['RunEGamma2022B','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
-workflows[140.005] = ['',['RunSingleMuon2022B','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
-
-
-workflows[140.015] = ['',['RunZeroBias2022C','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
-workflows[140.016] = ['',['RunJetMET2022C','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
-workflows[140.017] = ['',['RunEGamma2022C','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
-workflows[140.018] = ['',['RunMuon2022C','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
-workflows[140.019] = ['',['RunTau2022C','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
-workflows[140.020] = ['',['RunBTagMu2022C','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
-
-workflows[140.031] = ['',['RunZeroBias2022D','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
-workflows[140.032] = ['',['RunJetMET2022D','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
-workflows[140.033] = ['',['RunEGamma2022D','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
-workflows[140.034] = ['',['RunMuon2022D','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
-workflows[140.035] = ['',['RunTau2022D','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
-workflows[140.036] = ['',['RunBTagMu2022D','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
+
+workflows[140.001] = ['',['RunMinimumBias2022A','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
```

Comment:
hi, in light of the discussion going at https://its.cern.ch/jira/browse/PDMVRELVALS-155 I wonder if it's possible to profit of this PR to change the `RECONANORUN3` to include also standard DQM sequences:

https://github.com/cms-sw/cmssw/blob/6ba5904a9334bdc20ece0528a157efad0c20b699/Configuration/PyReleaseValidation/python/relval_steps.py#L2737

Reply:
Thanks, done.